src/main/java/jnr/ffi/LibraryLoader.java

```diff
@@ -474,8 +474,12 @@ private static void addPathsFromFile(Collection<String> paths, File file) {
                 in = new BufferedReader(new FileReader(file));
                 String line = in.readLine();
                 while (line != null) {
-                    // add even if doesn't exist! We're just adding the default paths, we check for validity elsewhere
-                    paths.add(line);
+                    // ignore empties, comments and include directives
+                    if (!line.isEmpty() && !line.trim().isEmpty()
```

Comment:
If we are going to `trim().isEmpty()` anyway we don't need the first check here.

Reply:
What's the purpose of this method?

Reply:
@headius Ah yes that's true, good eye!

@fzakaria If you mean the method `addPathsFromFile()` that was there before my changes, basically it adds all paths from a file, this is often `/etc/ld.so.conf` and/or the files located in `/etc/ld.so.conf.d/`.
In my Linux machine I would often get comments and empty lines, this is wasteful and can be ignored. There are `include` directives which I would like to add functionality for in the future since it can happen that a user/application may edit the master `/etc/ld.so.conf` file and add a new directory by doing `include $DIR` although this is rare as far as I'm aware